<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtPage</class>
 <widget class="QWidget" name="QtPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>642</width>
    <height>614</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QLabel" name="headerLabel">
     <property name="text">
      <string>&lt;b&gt;Configure Qt&lt;/b&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line12">
     <property name="frameShape">
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Qt Translations Directory</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="TextLabel1_2_2_5">
        <property name="text">
         <string>&lt;font color=&quot;#FF0000&quot;&gt;&lt;b&gt;Note:&lt;/b&gt; This setting is activated at the next startup of the application.&lt;/font&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="E5PathPicker" name="qt4TransPicker" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="toolTip">
         <string>Enter the path of the Qt translations directory.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="textLabel1_2_4">
        <property name="text">
         <string>&lt;b&gt;Note:&lt;/b&gt; Leave this entry empty to use an environment variable or the path compiled into the Qt library. Environment variables supported are
&lt;ul&gt;
&lt;li&gt;QT4TRANSLATIONSDIR for Qt4&lt;/li&gt;
&lt;li&gt;QT5TRANSLATIONSDIR for Qt5&lt;/li&gt;
&lt;li&gt;QTTRANSLATIONSDIR for any Qt variant&lt;/li&gt;
&lt;/ul&gt;</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>Qt Tools</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Tools Directory:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="E5PathPicker" name="qtToolsDirPicker" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="focusPolicy">
           <enum>Qt::StrongFocus</enum>
          </property>
          <property name="toolTip">
           <string>Enter the path of the Qt tools directory, if they are not found.</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>The tool executable is composed of the prefix, the tool name and the postfix. For win, the extension is added automatically.</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Qt-Prefix:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="qt4PrefixEdit">
          <property name="toolTip">
           <string>Enter the prefix for the Qt tools name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Qt-Postfix:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="qt4PostfixEdit">
          <property name="toolTip">
           <string>Enter the postfix for the Qt tools name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="qt4SampleLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>This gives an example of the complete tool name</string>
          </property>
          <property name="text">
           <string>designer</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>PyQt Tools</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Tools Directory:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="E5PathPicker" name="pyqtToolsDirPicker" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="toolTip">
         <string>Enter the path of the PyQt tools directory, if they are not found.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>pyuic / pyside-uic Options</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Indent Width:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="pyuicIndentSpinBox">
        <property name="toolTip">
         <string>Select the indent width (default: 4)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="minimum">
         <number>2</number>
        </property>
        <property name="maximum">
         <number>16</number>
        </property>
        <property name="value">
         <number>4</number>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>448</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="QCheckBox" name="pyuicImportsCheckBox">
        <property name="text">
         <string>Generate imports relative to '.'</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>496</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>E5PathPicker</class>
   <extends>QWidget</extends>
   <header>E5Gui/E5PathPicker.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>qt4TransPicker</tabstop>
  <tabstop>qtToolsDirPicker</tabstop>
  <tabstop>qt4PrefixEdit</tabstop>
  <tabstop>qt4PostfixEdit</tabstop>
  <tabstop>pyqtToolsDirPicker</tabstop>
  <tabstop>pyuicIndentSpinBox</tabstop>
  <tabstop>pyuicImportsCheckBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
